Docs/topics/vita.md

```diff
@@ -0,0 +1,92 @@
+Next, you want to identify the title ID of your game. This can be done in multiple ways. 
+- On the stem of the box, you'll see something along the lines of "PCSF00516" (4 letters followed by 5 numbers), this is your title ID. 
+- Look in `ux0:app/` (or wherever your Vita is configured to install games) for one of the following title IDs. If you see one of these items in that folder, then that is your title ID. 
+Note that this list is non-exhaustive, if your game's title ID is missing, please let us know!
+  - PCSA22018 - US Cartridge
+  - PCSA00017 - US Digital
+  - PCSA00549 - US Digital (Marvel Super Hero Edition)
+  - PCSF00021 - EU Cartridge
+  - PCSF00516 - EU Cartridge (Marvel Super Hero Edition)
+  - PCSA22106 - Canada Cartridge
+  - PCSD00006 - Asia Cartridge
+  - VCAS32010 - Asia Cartridge
+  - PCSC00013 - JP Digital
+  - VCJS10006 - JP Cartridge
+  - VCKS62003 - Korea Cartridge
+
+Now that you know your game's title ID, you can go on to the next step. On your Vita, 
+stop the FTP server and return to VitaShell's menu.  Then, navigate to `ur0:tai/` and select `config.txt`.
+
+With the file open, scroll to the bottom using the D-Pad, then press triangle, and select "Insert empty line". 
+On this new empty line, put an asterisk followed by **your** title ID, this should look something like `*PCSC00013`. 
+After you have placed that, make a new line again, and put the following text: `ur0:tai/Allefresher_vita.suprx`
+
+After you are done, you should see something like this, but with your title ID.
+
+![2024-05-04-210838.png](vitashell-after-config-edit.png)
+
+<warning>
+Make sure that you do not place the plugin's name under <code>*main</code> or <code>*KERNEL</code>, this could cause your Vita to not turn on. 
+If this happens, hold the <code>L</code> button on boot to disable any plugins, then fix the config in VitaShell.
+</warning>
+
+### Changing the server you are connecting to
+
+If you are connecting to the official LittleBigRefresh instance, you can ignore this segment, and skip to [Step 5](#step-5-finalize-the-setup).
+
+Allefresher Vita allows the user to specify another server to connect to, although it requires a bit more setup. 
+
+Allefresher will read from a text file named `allefresher.txt` from `ux0:` on your Vita to get the URL to connect with, 
+defaulting to the LittleBigRefresh instance if not found. 
+This file contains the patching URL for the server you want to connect to. You can get this URL from the server owner.
+
+However, for the game to be able to read this file, you need to install the [ioPlus](https://www.gamebrew.org/wiki/IoPlus_Vita) plugin.
+This is because the Vita disallows the game to access files outside its own directory, and Allefresher has the same permissions as the game itself.
+ioPlus expands the permissions of the game to allow read/write to arbitrary files, allowing Allefresher to find your config.
+
+## Step 5: Finalize the setup!
```

Comment:
Should mention that the user needs to have updated their game to the latest version (1.22). Maybe include a short guide on how to actually update, because I've seen people who don't understand the process.

If the game is updated, `Update History` will show up when pressing triangle on the livearea bubble, and it'll say `Version 1.22`.

Reply:
Saying to update in the latest release is said in the message [here](https://github.com/LittleBigRefresh/Docs/pull/16/files#diff-0fb261d7d26a7a49357c83ab8d7bd06e60e522204ddf56d51602699d689f2edbR3), but i'll add the blurb about double checking the version